nssp/delphi_nssp/run.py

```diff
@@ -90,7 +90,7 @@ def run_module(params):
         for geo in GEOS:
             df = df_pull.copy()
             df["val"] = df[signal]
-            logger.info("Generating signal and exporting to CSV", metric=signal)
+            logger.info("Generating signal and exporting to CSV", signal=signal)
```

Comment:
add `geo` like you did in nwss/run.py
```suggestion
            logger.info("Generating signal and exporting to CSV", geo=geo, signal=signal)
```